server/server.go

```diff
@@ -60,6 +61,17 @@ func StartServer() {
 		}
 	}()
 
+	// Add profiling flag to enable profiling routes.
+	if os.Getenv("PPROF_ENABLED") != "" {
```

Comment:
It must be noted that in this setup, if `PPROF_ENABLED` set to _any_ value, it will be enabled.